### PR TITLE
original dataset + recoded tv-features; r-script for recoding

### PR DESCRIPTION
**Pull request by rheuer83**
Here are the joined original and tv-rds and the script used to generate it...

### CODE REVIEW

File: recoding_tv.R [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+library(sjmisc)
+library(here)
+
+anes <- read_delim(here("data","anes_timeseries_2016_rawdata.txt"), delim = "|")
+
+clean <- readRDS("clean.rds")
+
+anes_trimmed <- anes %>% 
+  dplyr::select(V161364:V161427) %>%
+  rename("20/20" = V161364,
```
Comment (rrodriguezbarron) [inline recoding_tv.R:11]: @rheuer83 we should keep the same taxonomy: 

- All lower case
- Start with "tv_"
- remove all symbols, e.g. "/" in the 20/20; just change for another underscore ("_")

**Review by rrodriguezbarron — CHANGES_REQUESTED** [review]
Please review feedback on taxonomy of variables.

File: recoding_tv.R (hunk 1 above)
Comment (rheuer83) [reply recoding_tv.R:11]: This is already implemented in the code. (via clean_names from the janitor package)

Comment (rheuer83) [reply recoding_tv.R:11]: I have kept the original names in the initial recoding section for transparency. The clean_names function converts to snake-case and removes symbols...

Comment (rrodriguezbarron) [reply recoding_tv.R:11]: I had no idea you could do that! Let me go back and check it, then.

File: recoding_tv.R [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+         "Lo Mejor de Caso Cerrado" = V161420,
+         "Noticiero Enrique Gratas" = V161421,
+         "Noticiero Telemundo" = V161422,
+         "Noticiero Univision" = V161423,
+         "El Senor de los Cielos" = V161424,
+         "Que Noche! con Angelica y Raul" = V161425,
+         "Primer Impacto" = V161426,
+         "Cronicas de Sabado" = V161427) %>% 
+  rename_all( ~ paste0("tv_", .x)) %>% 
+  clean_names() %>%
```
Comment (rrodriguezbarron) [inline recoding_tv.R:76]: This is so cool. I didn't know we could do this.

**Review by rrodriguezbarron — CHANGES_REQUESTED** [review]
@rheuer83 is there anyway you can integrate this _within_ data.rds? That way we won't have to run two codes to get the data frame. Also, I propose we keep **clean.rds** as the main data set name.

File: Data.Rmd [hunk 3]
```diff
@@ -74,6 +138,7 @@ clean <- raw %>%
          ft_transgender_people = V162111,
          ft_scientists = V162112,
          ft_blm = V162113) %>% 
+  
   # Recoding ideology
   mutate(ideology = dplyr::recode(ideology,
```
Comment (rrodriguezbarron) [inline Data.Rmd:143]: This looks great.

**Review by rrodriguezbarron — COMMENTED** [review]
I don't know how to only merge the Data.Rmd and not merge the other files. Can you close this pull and only make one with the modificatios to Data.Rmd ?

**Pull request closed without merging**